lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1417,26 +1417,32 @@ sub register_assets_from_settings {
 
     # check assets and fix the file names
     for my $k (keys %assets) {
-        my $a = $assets{$k};
-        if ($a->{name} =~ /\//) {
-            log_info "not registering asset $a->{name} containing /";
+        my $asset = $assets{$k};
+        my ($name, $type) = ($asset->{name}, $asset->{type});
+        unless ($name && $type) {
+            log_info 'not registering asset with empty name or type';
```

Comment:
I would add the asset type here and upgrade the log level to error

Reply:
well, there is no type - that's the error (in case)

Reply:
I thought making a different if or (unless) for `$name` and `$type` would be a bit over-engineering for this hopefully rare error case.

Reply:
indeed it is, but actually my comment was about knowing the `type` of asset, rather than knowing the value of `$name`